docs/reference/src/main/asciidoc/decorators.asciidoc

```diff
@@ -181,8 +181,5 @@ Decorators enabled using `@Priority` are called before decorators
 enabled using `beans.xml`, the lower priority values are called first.
 
 NOTE: Having a decorator enabled by `@Priority` and in the same time listed
-in `beans.xml` leads to a non-portable behaviour! This combination of enablement 
-should therefore be avoided in order to maintain consistent behaviour across 
-different CDI implementations.
-As for Weld, such decorator is only invoked once in the `@Priority part of the 
+in `beans.xml`, such decorator is only invoked once in the `@Priority part of the
```

Comment:
This sentence sounds awkward to me (`...in the same time listed in beans.xml, such decorator ...`). How about:
`NOTE: A decorator enabled by @Priority and in the same time listed in beans.xml is only invoked once in the @Priority ...`
If you change this, please change the one in interceptors section as well.